packages/esm-patient-immunizations-app/src/immunizations/immunizations-form.component.tsx

```diff
@@ -79,17 +79,18 @@ const ImmunizationsForm: React.FC<DefaultWorkspaceProps> = ({ patientUuid, close
       ).then(
         (response) => {
           response.status === 201 && closeWorkspace();
-          showToast({
+          showSnackbar({
             kind: 'success',
-            description: t('vaccinationSaved', 'Vaccination saved successfully'),
+            title: t('vaccinationSaved', 'Vaccination saved successfully'),
+            isLowContrast: true,
           });
         },
         (err) => {
-          showNotification({
+          showSnackbar({
             title: t('errorSaving', 'Error saving vaccination'),
             kind: 'error',
-            critical: true,
-            description: err?.message,
+            isLowContrast: false,
```

Comment:
If we have low contrast for success and high for error, isn't it better to make it low contrast in the notification component itself?
```
lowContrast : props.lowContrast || props.kind !== 'error'
```
```

Reply:
I will try it and see if it works generically.